sources/platform/actors/running/actor_standby.md

```diff
@@ -60,11 +60,83 @@ However, running Actors in Standby mode might have unexpected costs, as the Acto
 
 No, even if you use the Actor-level hostname with the default configuration, the background Actor runs for your requests are not shared with other users.
 
-#### How can I enable Standby for my Actor
+#### How can I develop Actors using Standby mode
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 You can head to the Settings tab of your Actor, enable Standby mode, and set the default configuration.
 ![Standby for creators](./images/actor_standby/standby-creators.png)
 
+For the coding part, you can use Apify SDK to retrieve the port Actor Standby should be listening on.
+
+<Tabs>
+<TabItem value="JavaScript" label="JavaScript">
+
+```js
+import { Actor } from 'apify';
+
+await Actor.init();
+console.log(Actor.config.get('standbyPort'));
+```
+
+</TabItem>
+<TabItem value="Python" label="Python">
+
+```python
+from apify import Actor, Configuration
+
+async def main() -> None:
+    async with Actor:
+        config = Configuration.get_global_configuration()
+        print(config.standby_port)
+```
+
+</TabItem>
+</Tabs>
```

Comment:
I would skip this and go straight for the example with the HTTP server, it's not too complicated, so there's no need to split it into two examples.

Reply:
Done